vendor/bat-native-ledger/src/bat/ledger/internal/wallet/wallet_create.h

```diff
@@ -21,20 +18,20 @@ namespace wallet {
 
 class WalletCreate {
  public:
-  explicit WalletCreate(LedgerImpl* ledger);
-  ~WalletCreate();
+  explicit WalletCreate(LedgerImpl*);
 
-  void Start(ledger::ResultCallback callback);
+  void CreateWallet(ResultCallback, absl::optional<std::string>&& geo_country);
```

Comment:
Generally, the style is to have the callback go after the args? We should probably stick with that convention unless we plan on moving the codebase to that pattern.

Reply:
Yeah, good point, done.